pipettes/firmware_l5/main.cpp

```diff
@@ -69,6 +69,21 @@ class EEPromHardwareIface : public eeprom::hardware_iface::EEPromHardwareIface {
         }
     }
 };
+
+auto convert_to_motor_hardware(
+    pipette_motor_hardware::HardwareConfig motor_config)
```

Comment:
this strikes me as slightly worrying but I'm not sure why - why does this need to exist? can `convert_to_motor_hardware` be templated or something?

Reply:
It shouldn't exist. There was a typo on my part awhile back where the hardware configs were accidentally using the pipette hardware configs.

The way that we're doing tip sensing is going to change so the need for a pipette hardware class seems moot to me anyways.

I didn't want to address that in this PR since it's already a large-ish refactor. I'll make a ticket to track the work.